buildenv/jenkins/openjdk_aarch64_linux_xl

```diff
@@ -0,0 +1,14 @@
+#!groovy
+LABEL='ci.role.test&&sw.os.linux&&hw.arch.aarch64'
+
+stage('Queue') {
+    node((params.LABEL) ? params.LABEL : LABEL) {
+        PLATFORM = 'aarch64_linux_largeHeap'
+        SDK_RESOURCE = 'upstream'
+        SPEC='linux_aarch64_cmprssptrs'
```

Comment:
if this is for large heap, then `cmprssptrs` should be removed from `SPEC`

Reply:
Good catch. Fixed